WcaOnRails/app/models/registration.rb

```diff
@@ -204,13 +204,14 @@ def to_wcif(authorized: false)
     {
       "wcaRegistrationId" => id,
       "eventIds" => events.map(&:id).sort,
-      "status" => if accepted?
+      "status" => if accepted? || !is_competing?
```

Comment:
Might be worth a comment, see above. On second thought, might even be worth a refactor into its own method in `Registration`, along the lines of `attending?`

Reply:
Moved into its own method.